src/css/utilities/_variables.scss

```diff
@@ -0,0 +1,36 @@
+// Variables
+
+// Declaration of common measurements for margin and padding
+
+//$m-0:0rem
+//$m-1:0.5rem
+//$m-2:1rem
+//$m-3:1.5rem
+//$m-4:2rem
+//$m-5:2.5rem
+//$m-6:3rem
+//$m-7:3.5rem
+//$m-8:4rem
+//$m-9:4.5rem
+//$m-10:5rem
+//$m-11:5.5rem
+//$m-12:6rem
+
+// margin
+$m-0: 0;
+$m-1: 0.5rem;
+$m-2: 1rem;
+$m-3: 1.5rem;
+$m-4: 2rem;
+$m-5: 2.5rem;
+$m-6: 3rem;
+$m-7: 3.5rem;
+$m-8: 4rem;
+$m-9: 4.5rem;
+$m-10: 5rem;
+$m-11: 5.5rem;
+$m-12: 6rem;
```

Comment:
I was also thinking if we could have something more like:

```
$m-0: 0;
$m-xxs: 0.5rem;
$m-xs: 1rem;
$m-s: 1.5rem;
$m-m: 2rem;
$m-l: 3rem;
$m-xl: 4rem;
$m-xl: 6rem;
```
Since it seems like the rest of the variables are never used and changing the naming of the variables removes the need to have every 0.5 step measure defined. @ErriGarcia thoughts?